tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>"We can also build extraction regions by supplying coordinates defined in an <code>astropy.coordinates.SkyCoord</code>  to  <code>pvextractor.Path</code> "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>